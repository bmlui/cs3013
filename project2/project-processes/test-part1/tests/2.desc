evaluate pwd builtin, must use chdir() 
