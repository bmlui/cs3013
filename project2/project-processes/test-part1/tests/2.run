grep "chdir(" ../wshell.c -l; PATH="/bad" ./wshell < tests/2.in
