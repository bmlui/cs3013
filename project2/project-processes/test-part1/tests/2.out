../wshell.c
test-part1$ cd /
/$ pwd
/
/$ exit
